src/app/asset-page/artstor-viewer/artstor-viewer.component.scss

```diff
@@ -116,24 +107,20 @@ $desktop: 992px;*/
 
 
 .asset-viewer__edge-arrows {
-    opacity: 0;
     transition: opacity 0.2s linear;
     position: absolute;
     pointer-events: none;
     top: calc(50% - 120px);
     width: 100%;
 }
 
-.asset-viewer:hover .asset-viewer__edge-arrows {
-    opacity: 1;
-}
-
 .asset-viewer__edge-arrows .btn {
     pointer-events: all;
     width: 42px;
     height: 42px;
     border-radius: 50%;
     background: rgba(66,66,66,0.6);
+    z-index: 9999;
```

Comment:
For whatever reason moving the viewer buttons before the iiif viewer causes the pagination buttons to be covered up. Adding a z-index here ensures they are overlaid on top of the viewer